utils/getenv/getenv.go

```diff
@@ -0,0 +1,18 @@
+package getenv
```

Comment:
nit: let's choose a different name to make this more extensible in the future. You can just create a file called `utils.go` in the `utils` directory and then import it with `"github.com/aws/amazon-vpc-cni-k8s/utils"`